### ISSUE TITLE
Register a URL handler scheme

### ISSUE THREAD

**Issue by christefano** [JadenGeller/Helium#282]
I use Choosy (choosyosx.com) to open site-specific URLs in various browser apps, and I’d love to use Helium in this way. In other words, I click on a YouTube link and Choosy, as my system’s default web browser, offers to open it in Helium.

Right now it doesn’t seem possible. Choosy opens Helium, yet Helium doesn’t seem to be getting the message. I didn’t see any issues for this already. So here it is! I did see an issue for adding support for arguments via the command line, so I guess I could whip up an AppleScript or something… That would be slower than native URL handling support, tho.
 
tl;dr: Can Helium receive URLs other than by command line?

**Comment by slashlos** [comment]
AFAIK, using helium: as the scheme should work?

**Comment by Kaishu-Sahu** [comment]
I am also facing the same issue. I use OpenIn, which shows you a list of available browsers to open a link but Helium is not there. @slashlos Right now, OpenIn doesn't allow to have schema level customization it can give normal CLI arguments to the app.

**Comment by slashlos** [comment]
Well, that's odd as AFAIK, and tested just now (Helium 2.0 with macOS 11.3.1), Helium does handle its **helium:** scheme.

The question as to a default browser are, IMHO, a bridge too far for what Helium does currently. But if it's how to get a URL directly to Helium (any version) why not just drag the URL onto the app icon or am I missing something?
